doc/about.rst

```diff
@@ -96,86 +96,176 @@ following paper:
+   :target: https://www.intel.com/
 
-`Columbia University <https://columbia.edu/>`_ funds Andreas Müller since 2016.
+.. figure:: images/nvidia.png
+   :width: 70pt
+   :align: center
+   :target: https://www.nvidia.com/
 
-.. image:: themes/scikit-learn/static/img/columbia.png
-   :width: 100pt
+**Silver Members**
+
+.. figure:: images/dataiku.png
+   :width: 70pt
+   :align: center
+   :target: https://www.dataiku.com/
+
+........  
+
+.. figure:: themes/scikit-learn/static/img/columbia.png 
+   :width: 50pt
    :align: center
    :target: https://www.columbia.edu/
 
-Andreas Müller also received a grant to improve scikit-learn from the `Alfred P. Sloan Foundation <https://sloan.org>`_ in 2017.
+   `Columbia University <https://columbia.edu/>`_ funds Andreas Müller since 2016
+
+..........
 
-.. image:: images/sloan_banner.png
-   :width: 200pt
+.. figure:: images/sloan_banner.png
+   :width: 100pt
    :align: center
    :target: https://sloan.org/
 
-`The University of Sydney <https://sydney.edu.au/>`_ funds Joel Nothman since July 2017.
+   Andreas Müller received a grant to improve scikit-learn from the
+   `Alfred P. Sloan Foundation <https://sloan.org>`_ .
+   This grant supports the position of Nicolas Hug and Thomas J. Fan.
 
-.. image:: themes/scikit-learn/static/img/sydney-primary.jpeg
-   :width: 200pt
+...........
+
+.. figure:: themes/scikit-learn/static/img/sydney-primary.jpeg
+   :width: 100pt
    :align: center
    :target: https://sydney.edu.au/
 
-`The Labex DigiCosme <https://digicosme.lri.fr>`_ funded Nicolas Goix (2015-2016),
-Tom Dupré la Tour (2015-2016 and 2017-2018), Mathurin Massias (2018-2019) to work part time
-on scikit-learn during their PhDs. It also funded a scikit-learn coding sprint in 2015.
+   `The University of Sydney <https://sydney.edu.au/>`_ funds Joel Nothman since
+   July 2017.
 
-.. image:: themes/scikit-learn/static/img/digicosme.png
-   :width: 200pt
+............
+
+.. figure:: images/anaconda.png
+   :width: 100pt
+   :align: center
+   :target: https://sydney.edu.au/
+
+   `Anaconda, Inc <https://www.anaconda.com/>`_ funds Adrin Jalali since .
+
+
+
+Past Sponsors
+.............
+
+.. figure:: images/inria-logo.jpg
+   :width: 100pt
+   :align: center
+   :target: https://www.inria.fr
+
+   `INRIA <https://www.inria.fr>`_ actively supports this project. It has
+   provided funding for Fabian Pedregosa (2010-2012), Jaques Grobler
+   (2012-2013) and Olivier Grisel (2013-2017) to work on this project
+   full-time. It also hosts coding sprints and other events.
+
+.....................
+
+.. figure:: images/cds-logo.png
+   :width: 100pt
+   :align: center
+   :target: https://www.datascience-paris-saclay.fr/
+
+   `Paris-Saclay Center for Data Science
+   <https://www.datascience-paris-saclay.fr/>`_
+   funded one year for a developer to work on the project full-time
+   (2014-2015), 50% of the time of Guillaume Lemaitre (2016-2017) and the
+   position of Joris van den Bossche (????-????).
```

Comment:
Hello @jorisvandenbossche , could you please confirm the extent of your contract and the type (full or part time?)? Thanks!

Reply:
You can put the same as for Guillaume, but with years 2017-2018